tiledb/cloud/compute/delayed.py

```diff
@@ -170,6 +172,39 @@ def __call__(self, *args, **kwargs):
         return self
 
 
+class DelayedMultiArrayUDF(DelayedBase):
+    def __init__(
+        self,
+        func: Union[str, Callable, None] = None,
```

Comment:
Neither of these should have a default value, since they both need to be set.

Reply:
@thetorpedodog I took this from the underlying function, should we change it there too?

Reply:
No, that implements support for the legacy `name` argument, when you could pass the function as `name='some_registered_udf'`, but this is not supported in the Delayed system.

Reply:
I meant from `exec_multi_array_udf_base`

Reply:
That’s what I’m referring to.

Reply:
Thanks for clarifying!